src/api.js

```diff
@@ -150,6 +150,8 @@ export default class FacebookAdsApi {
             ${Object.keys(data).length > 0 ? JSON.stringify(data) : ''}`,
           );
         }
+        console.error(`Error in ${method} request to ${url} with data: ${JSON.stringify(data)}`);
+        console.error('Response:', response);
```

Comment:
could you remove these two lines of changes? the api.js file is heavily used and could have bigger impact.